analyses/stackanalyses/controller.go

```diff
@@ -195,3 +198,52 @@ func (mc *Controller) getManifestName(manifestFile string) string {
 	}
 	return stats.Name()
 }
+
+// ProcessSummary processes summary results, return true if Vul found
+func ProcessSummary(analysedResult driver.GetResponseType, jsonOut bool) bool {
+	out := summary.GetResultSummary(analysedResult)
+	if jsonOut {
+		outputSummaryJSON(out)
+	} else {
+		outputSummaryPlain(out)
+	}
+	if out.TotalVulnerabilities > 0 {
+		// If Vulnerability found, return true
+		return true
+	}
+	return false
+}
+
+// outputSummaryJSON stdout analyses summary output as JSON
+func outputSummaryJSON(result *summary.StackSummary) {
+	b, err := json.MarshalIndent(result, "", "\t")
+	if err != nil {
+		log.Fatal().Msg("Error forming CLI JSON Response.")
+	}
+	fmt.Fprintln(os.Stdout, string(b))
+}
+
+// outputSummaryPlain stdout analyses summary output as JSON
+func outputSummaryPlain(result *summary.StackSummary) {
+
+	yellow := color.New(color.FgHiYellow, color.Bold).SprintFunc()
+	white := color.New(color.FgHiWhite, color.Bold).SprintFunc()
+	red := color.New(color.FgHiRed, color.Bold).SprintFunc()
+	blue := color.New(color.FgHiBlue, color.Bold).SprintFunc()
+	magenta := color.New(color.FgHiMagenta, color.Bold).SprintFunc()
+	fmt.Println("Powered by Snyk")
+	fmt.Println("\nSummary Report for Analyses:")
```

Comment:
It is not a common practice to add \n in the front. It hinders readability too.
```suggestion
	fmt.Println("Powered by Snyk\n")
	fmt.Println("Summary Report for Analyses:")
```

Reply:
Ack.